### DISCUSSION TITLE
Support sx1262/1268

### DISCUSSION THREAD

**Discussion by StreakingJerry** [ExpressLRS/ExpressLRS#2921] [Ideas]
The new sx1262/1268 is muck better than sx1272. Will the modual such as Ra-01s and E22-400M30S be supported? (they are 433 mhz which can be even longer range and won't confilct with cellphone network band 8)

**Comment by JyeSmith** [comment]
In all testing the SX1262/LLCC68 under performed compared to the SX1276.  There is an extensive Semtech forum post on the issues, which is now only available via Internet Archive.  So it will not be supported as the LR1121 outperforms everything!

https://web.archive.org/web/20240310140117/https://forum.lora-developers.semtech.com/t/llcc68-poor-sensitivity/1480

433 is also still under testing.  All testing to date for the SX127, SX1262, LR1121 have not shown it to out perform 900.  But this is still an area of active development.
